src/drawing/__tests__/DrawingAnnotations-test.tsx

```diff
@@ -163,18 +153,18 @@ describe('DrawingAnnotations', () => {
             expect(wrapper.exists(PopupDrawingToolbar)).toBe(false);
         });
 
-        test('should apply ba-is-faded if is currently drawing', () => {
-            const wrapper = getWrapper({ isCreating: true, drawnPathGroups: pathGroups });
+        test('should apply ba-is-drawing if is currently drawing', () => {
```

Comment:
We can probably omit the specific class from the test description to avoid it going stale.